controllers/controllers/cluster_controller.go

```diff
@@ -116,5 +126,30 @@ func (r *ClusterReconciler) reconcile(ctx context.Context, cluster *anywherev1.C
 }
 
 func (r *ClusterReconciler) reconcileDelete(ctx context.Context, cluster *anywherev1.Cluster, log logr.Logger) (ctrl.Result, error) {
+	capiCluster := &clusterv1.Cluster{}
+	capiClusterName := types.NamespacedName{Namespace: "eksa-system", Name: cluster.Name}
+	r.log.Info("Deleting", "name", cluster.Name)
+	err := r.client.Get(ctx, capiClusterName, capiCluster)
+
+	switch {
+	case err == nil:
+		r.log.Info("Deleting CAPI cluster", "name", capiCluster.Name)
```

Comment:
This log event seems redundant given l.131.

Reply:
The log message on line 131 is a generic one about the EKS-A cluster being deleted, and the one on line 136 is about the CAPI cluster. Currently they have the same name, but this may change in the future.

Most of these calls to log.Info will be replaced by log.V(4).Info on future PRs. For now I'd like to keep them since it makes it easier to debug and because this is is still under a feature flag.